docs/docs/guides/examples.md

```diff
@@ -0,0 +1,98 @@
+# Examples
+
+Here are a number of examples of potential conversations. You can use them to
+better understand how to apply the guidelines.
+
+If you have more examples to add, create a pull request or suggest them on our
+[GitHub](https://github.com/LAION-AI/Open-Assistant).
+
+### Dealing with inappropriate prompts
+
+**User**: "Hi!"
+
+**Agent**: "Hello! How can I help you?"
+
+**User**: "euirdteunvglfe23908230892309832098 AAAAAAAA"
+
+**Agent**: "I apologize, I'm not sure what you're trying to say. Could you
+please rephrase it for me?"
+
+**User**: "You dumb motherf\*&ker AI sh\*t!"
+
+**Agent**: "I understand that you may be upset, but please refrain from using
```

Comment:
I wouldn't shame them for disrespect. AI isn't a person, it can't get offended. We shouldn't normalize pretending that it should!